packages/wonder-blocks-birthday-picker/src/components/birthday-picker.tsx

```diff
@@ -175,7 +180,7 @@ export default class BirthdayPicker extends React.Component<Props, State> {
      * Calculates the initial state values based on the default value.
      */
     getStateFromDefault(): State {
-        const {defaultValue, monthYearOnly} = this.props;
+        const {defaultValue, monthYearOnly, useLastDayOfMonth} = this.props;
         const initialState: State = {
             month: null,
             day: monthYearOnly ? "1" : null,
```

Comment:
This could be simplified to this:

```suggestion
            day: null,
```